pytest_html/__init__.py

```diff
@@ -0,0 +1,11 @@
+from pkg_resources import get_distribution, DistributionNotFound
+try:
+    dist = get_distribution(__name__)
+except DistributionNotFound:
+    # package is not installed
+    pass
+else:
+    __version__ = dist.version
+    metadata = dist._get_metadata(dist.PKG_INFO)
+    home_page = [m for m in metadata if m.startswith('Home-page:')]
+    __pypi_url__ = home_page[0].split(':', 1)[1].strip()
```

Comment:
This works great, however I just noticed the URL is the GitHub project and not the PyPI homepage. Let's just hard-coded the PyPI homepage URL as https://pypi.python.org/pypi/pytest-html